tests/client/client_test.go

```diff
@@ -617,38 +634,41 @@ type testClientSuite struct {
 	reportBucket    pdpb.PD_ReportBucketsClient
 }
 
-func checkerWithNilAssert(c *C) *assertutil.Checker {
-	checker := assertutil.NewChecker(c.FailNow)
+func checkerWithNilAssert(re *require.Assertions) *assertutil.Checker {
+	checker := assertutil.NewChecker(func() {
+		re.FailNow("should be nil")
+	})
 	checker.IsNil = func(obtained interface{}) {
-		c.Assert(obtained, IsNil)
+		re.Nil(obtained)
 	}
 	return checker
 }
 
-func (s *testClientSuite) SetUpSuite(c *C) {
+func newTestClient(re *require.Assertions) *testClient {
```

Comment:
We can also use suite here.

Reply:
The tests inside a suite can not be run in parallel, see https://github.com/stretchr/testify/issues/187.

Reply:
After consideration, let's keep using suite here to finish the migration first.